lib/ecco/row_event_listener.rb

```diff
@@ -26,10 +27,7 @@ def on_event(event)
       when table_event
         @table_map_event = event
       when *accepted_events
-        row_event          = Ecco::RowEvent.new
-        row_event.table_id = data.get_table_id
-        row_event.rows     = data.rows
-        row_event.type     = row_type_to_string(type)
+        row_event = row_event_create_with(type, data)
```

Comment:
a bit related to what I wrote in https://github.com/twingly/ecco/pull/56/files#r514231709, just try to tackle `row_event.rows` and nothing else in the first "iteration" of changing things in order to keep the scope down a bit